core/src/test/scala/org/apache/spark/ui/UISuite.scala

```diff
@@ -179,16 +179,15 @@ class UISuite extends SparkFunSuite {
   test("verify appUIAddress contains the scheme") {
```

Comment:
What do you think about renaming `spark.driver.appUIAddress` to `spark.driver.webUrl`? Not necessarily important, but merely for the sake of consistency...